docroot/modules/contrib/ova/src/Plugin/Field/FieldFormatter/TextAudioFormatter.php

```diff
@@ -0,0 +1,234 @@
+    ];
+    $element['muted'] = [
+      '#title' => t('Muted'),
+      '#type' => 'checkbox',
+      '#default_value' => $this->getSetting('muted'),
+    ];
+    $element['annotations'] = [
+      '#title' => t('Enable Annotations'),
+      '#type' => 'checkbox',
+      '#default_value' => $this->getSetting('annotations'),
+      '#description' => t('Enable Annotation_store Module'),
+    ];
+    $element['extensions'] = [
+      '#title' => t('Allow Video Formats'),
+      '#type' => 'textarea',
+      '#default_value' => $this->getSetting('extensions'),
+    ];
+    $element['default_image'] = [
+      '#title' => t('Default Image'),
+      '#type' => 'textfield',
+      '#default_value' => $this->getSetting('default_image'),
+    ];
+    $element['preload'] = [
+      '#title' => t('Preload'),
+      '#type' => 'select',
+      '#default_value' => $this->getSetting('preload'),
+      '#options' => array(
+        'none' => 'none',
+        'metadata' => 'metadata',
+        'auto' => 'auto',
+      ),
+      '#description' => t('Hint to the browser about whether optimistic downloading of the video itself or its metadata is considered worthwhile.'),
+    ];
+    return $element;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsSummary() {
+    $summary = array();
+    $summary[] = t('HTML5 Video (@widthx@height@controls@autoplay@loop@muted@annotations@extensions@default_image).', [
+      '@width' => $this->getSetting('width'),
+      '@height' => $this->getSetting('height'),
+      '@controls' => $this->getSetting('controls') ? t(', controls') : '' ,
+      '@autoplay' => $this->getSetting('autoplay') ? t(', autoplaying') : '' ,
+      '@loop' => $this->getSetting('loop') ? t(', looping') : '' ,
+      '@muted' => $this->getSetting('muted') ? t(', muted') : '',
+      '@annotations' => $this->getSetting('annotations') ? t(', annotations') : '',
+      '@extensions' => $this->getSetting('extensions') ? t(', extensions') : '',
+      '@default_image' => $this->getSetting('default_image') ? t(', default_image') : '',
+    ]);
+    return $summary;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    global $base_url;
+    $elements = array();
+    $video_items = array();
+    $ext = 'mp3';
+    foreach ($items as $delta => $item) {
+      $video_uri = $item->getValue();
+      $video_items[] = Url::fromUri(file_create_url($video_uri['value']));
+      $var = $this->getSettings();
+      $extract_extensions = explode(',', $var['extensions']);
+      foreach ($extract_extensions as $key => $value) {
+        if (strpos($video_uri['value'], $value)) {
+          $ext = $value;
+        }
+        else {
+          $ext = 'mp3';
+        }
+      }
+    }
+    $default_image_path = $base_url . '/modules/contrib/ova/audio.jpg';
+    if (isset($video_uri['value']) && ($this->getSetting('annotations') == 0)) {
+      $elements[] = array(
```

Comment:
The elements array is similar for both if & else conditions. Would be better to initialize the variable with default values outside the conditional & then update the array indexes only inside the conditional.